### ISSUE TITLE
Address "ls -s" and "ls -S" confusion/differences

### ISSUE THREAD

**Issue by ndporter** [swcarpentry/shell-novice#1351]
The [General Syntax of a Shell Command section](https://swcarpentry.github.io/shell-novice/02-filedir/index.html#general-syntax-of-a-shell-command) uses `ls -s` and `ls -S` as examples of case differences in options.

Unfortunately, both commands provide different output for the same data based on OS and terminal platforms. This is recognized for `-s` with a note below the output that memory blocks are different between systems.

But `-S` also sorts differently because on Mac folder size for sorting includes the files contained in the folder (this matches `ls -l` but not `ls -s`), whereas on Windows, folders are still treated as size 0.

If possible, it might be better to use a different example of case difference or just leave it out other than a note. While we could certainly add explanations with more detail on both, using examples that have dramatically different output for what seems like it should be the same is more confusing than helpful for a summary section for novice learners.

**Comment by bkmgit** [comment]
Thanks for the suggestion. Learners should probably be alerted that there may be differences in some of the commands since they may use a shell on a remote computer and find a few differences. Examining the output of `ls --help` or `man ls` does show that the commands are case sensitive. Maybe comparing `ls -f /` and `ls -F /` maybe better?

**Comment by ndporter** [comment]
@bkmgit yes, -f/-F makes more sense to me than what's there as far as cognitive load. They won't always get identical output but if they followed the lesson exactly, the output should be the same across platforms - so you wouldn't need either the existing note on -s or another note on -S.

**Comment by 7anooch** [comment]
Is anyone planning on opening a PR regarding this point? If not I might. 

Also finding that `ls -s` unnecessarily confusing given the output is different on different filesystems (also its not a particularly helpful/intuitive command? `ls -lh` is a much better option for getting file sizes but its probably not the right point in the lesson to instroduce `ls -l`). I do like the alternative of trying `ls -f` and 'ls -F` given than `ls -F` is already introduced earlier

**Comment by bkmgit** [comment]
@7anooch Thanks, a PR is welcome.
